spec/models/spree_amazon/response_spec.rb

```diff
@@ -368,6 +367,48 @@
           end
         end
       end
+      
+      describe '#soft_decline?' do
+        context 'it is null' do
+          it 'return true' do
+            stub_auth_request(
+              order: order,
+              auth_ref: 'AUTH_REF',
+              status: 200,
+              body: build_mws_auth_declined_response(order: order, auth_ref: 'AUTH_REF', soft_decline: nil)
+            )
+            auth_response = SpreeAmazon::Response::Authorization.new(response)
+
+            expect(auth_response.soft_decline?).to eq(true)
+          end
+        end
+        context 'it is true' do
+          it 'return true' do
+            stub_auth_request(
+              order: order,
+              auth_ref: 'AUTH_REF',
+              status: 200,
+              body: build_mws_auth_declined_response(order: order, auth_ref: 'AUTH_REF', soft_decline: nil, soft_decline: true)
+            )
+            auth_response = SpreeAmazon::Response::Authorization.new(response)
+
+            expect(auth_response.soft_decline?).to eq(true)
+          end
+        end
+        context 'it is false' do
```

Comment:
empty line missing. See sublime linter: https://github.com/SublimeLinter/SublimeLinter-rubocop/issues/38 or if you don't use sublime, then just install Rubocop